sdcm/utils/sstable/sstable_utils.py

```diff
@@ -83,9 +88,8 @@ def is_sstable_encrypted(self, sstables=None) -> list:  # pylint: disable=too-ma
                     sstables_encrypted_mapping[sstable] = False
                 else:
                     scylla_metadata = json.loads(sstables_res.stdout)['sstables']
-                    sstables_encrypted_mapping[sstable] = \
-                        all('scylla_encryption_options' in metadata.get('extension_attributes', {})
-                            for table, metadata in scylla_metadata.items())
+                    sstables_encrypted_mapping[sstable] = all('scylla_encryption_options' in metadata.get('extension_attributes', {})
```

Comment:
Please, use this line state in the original commit of implementation in this PR.

Reply:
Not sure what you meant

Reply:
This line gets added in one commit and updated in another without real reason.
I propose to use the latest state in the first commit so second one would not change this line at all.

Reply:
fixed